### ISSUE TITLE
Using JS import statement

### ISSUE THREAD

**Issue by jimmck** [chinchang/web-maker#157]
Hi,
It seems I cannot use the JS import statement? I have added babel libs and selected babel in settings for JS. But console keeps showing import as bad keyword. I can include the CDN and access the code libs that way. Got to dig around to get stuff working like...

```
 components: {
    // <my-component> will only be available in parent's template
    Multiselect: VueMultiselect.Multiselect
  },
    data: {
      dataVar...
```
Just wondering if I am missing something or may not possible in current release.

Great Stuff!!!!

Regards
Jim

**Comment by chinchang** [comment]
@jimmck `import` statements require you to specify the file name from where something has to be imported. But in Web Maker, there is no concept of files that the compiler can look into to find the required module. Hence, I am not sure if it can work in Web Maker, and for that matter inside browser.

Will check more on this.

**Comment by jimmck** [comment]
Hi Kushagra
Thank You for the response. I thought might be an issue. Really Love the App. JsFiddle and Codepen are nice. The local capability to experiment with code is wonderful. I am using it everyday! Just saw your article on Templates gonna give that a try.

Regards 
Jim

**Comment by jimmck** [comment]
Hi Again Kushagra,
Follow up. What exactly are the Babel JS options and importing Babel CDN libraries used for? I am now assuming that Chrome JS support is the overriding factor in what what your 'compiler' checks when a user enters code? Any code referenced in the CDN must be either "packed" or native to Chromes JS support?

How about CSS? is that pre-processed at all? I see SASS is listed?

Just questioning the Workflow of code and execution.

Again Many Thanks for a Useful Tool!
Jim

**Comment by chinchang** [comment]
**How about CSS? is that pre-processed at all? I see SASS is listed?**

Yes. All preprocessors available in Web Maker work because they are available as browser standalone versions also. Eg. SASS compiler can compile SASS->CSS in the browser itself.

**Comment by chinchang** [comment]
`imports` now work in the "Files mode" which allows you to create files. This is available in 4.0 on new website -> https://webmaker.app

**Comment by jimmck** [comment]
Hi
Wonderful!! Thanks!
Jim
